### PR TITLE
For compareECL: Add option to accept extra keywords for both runs

### PR DESCRIPTION
**Pull request by lisajulia**
The extra keywords are printed out

### CODE REVIEW

File: test_util/EclRegressionTest.cpp [hunk 1]
```diff
@@ -276,32 +276,40 @@ void ECLRegressionTest::compareKeywords(const std::vector<std::string> &keywords
...
-        if (keywords2.size() > keywords1.size()) {
+        if (keywords2.size() > keywords1.size() - extraKeywordsFirstFile) {
             std::cout << "\nExtra keywords ("
-                      << std::to_string(keywords2.size() - keywords1.size())
+                      << std::to_string(keywords2.size() - keywords1.size() + extraKeywordsFirstFile)
                       << ") accepted in second file " << std::endl;
         }
+        if (extraKeywordsFirstFile > 0) {
+            std::cout << "\nExtra keywords ("
+                      << std::to_string(extraKeywordsFirstFile)
```
Comment (akva2) [inline test_util/EclRegressionTest.cpp:305]: stream can handle integers fine.

**Review by akva2 — APPROVED** [review]
Looks good, one very minor.

**Comment by lisajulia** [comment]
jenkins build this please

**Comment by lisajulia** [comment]
@blattms: Can you have another look here? Thanks!

**Comment by blattms** [comment]
Please also update the man-page [docs/man1/convertECL.1](/OPM/opm-common/blob/c908561d7f01a7b6c87dd71563644e8f3713b526/docs/man1/convertECL.1)

**Comment by lisajulia** [comment]
> Please also update the man-page [docs/man1/convertECL.1](/OPM/opm-common/blob/c908561d7f01a7b6c87dd71563644e8f3713b526/docs/man1/convertECL.1)

Updated :heavy_check_mark:

**Comment by blattms** [comment]
jenkins build this please

**Pull request merged**